composeApp/src/androidMain/kotlin/io/github/couchtracker/db/tmdb/Adapters.kt

```diff
@@ -0,0 +1,21 @@
+package io.github.couchtracker.db.tmdb
+
+import app.cash.sqldelight.ColumnAdapter
+import app.moviebase.tmdb.model.TmdbMovieDetail
+import app.moviebase.tmdb.model.TmdbReleaseDates
+import io.github.couchtracker.tmdb.TmdbLanguage
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+
+val tmdbLanguageAdapter = object : ColumnAdapter<TmdbLanguage, String> {
```

Comment:
Question: why aren't we using an `object <name> : ColumnAdapter<...>`? Also in this case they are constants, so they should use SCREAMING_SNAKE_CASE.